File: rest/authorized-connect-apps/list-get-example-1/list-get-example-1.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var authorizedApps = twilio.ListAuthorizedConnectApps(null, null);

    foreach (var authorizedApp in authorizedApps.AuthorizedConnectApps)
    {
      Console.WriteLine(authorizedApp.HomepageUrl);
    }
  }
}

File: rest/authorized-connect-apps/list-get-example-1/list-get-example-1.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over authorized_apps and print out a property for each one
@client.account.authorized_connect_apps.list.each do |authorized_app|
  puts authorized_app.connect_app_homepage_url
end
